### PR TITLE
Schedule updates

### PR DESCRIPTION
**Pull request by tracymiranda**

### CODE REVIEW

**Comment by tracymiranda** [comment]
@AlCutter @mhutchinson this pr includes the witness talk ptal & tag anyone else needed for review.

**Review by mhutchinson — APPROVED** [review]

File: content/schedule/registration.md [hunk 1]
```diff
@@ -16,4 +16,10 @@ A registration desk will be set up at the reception of [Google London - 6 Pancra
 Enjoy some coffee and breakfast and get to know your fellow conference attendees ahead of the talk sessions. 
 Please be aware that attendees will be required to pick up a new badge each day they attend the conference. 
 
+*Breakfast* 
+   * Freshly brewed filter coffee, a selection of Clipper tea infusions 
+   * Pressed orange juice
+   * Selection of pastries and preserves 
+
```
Comment (AlCutter) [inline content/schedule/registration.md:23]: No full English?! :)

File: content/schedule/talk10.md [hunk 2]
```diff
@@ -1,22 +1,20 @@
...
+speaker: Chervine Majeri
+speakerTitle: Research Engineer, Taurus
 ---
 
 <div class="font-google font-medium">
 
-
-TBC
-
+Highly sensitive code managing crypto secrets needs to run in secure environments, often very constrained in capacity. Given these constraints, I will explain how we managed to fit as much logic as possible, as close to the secrets as possible, while also addressing the  roles that transparency and verifiability play in ensuring security and trust.
```
Comment (AlCutter) [inline content/schedule/talk10.md:15]:
```suggestion
Highly sensitive code managing crypto secrets needs to run in secure environments, often very constrained in capacity. Given these constraints, I will explain how we managed to fit as much logic as possible, as close to the secrets as possible, while also addressing the roles that transparency and verifiability play in ensuring security and trust.
```

File: content/schedule/witness.md [hunk 3]
```diff
@@ -1,23 +1,36 @@
...
 url: schedule/witness.html
 layout: summittalk
-title: The Witness Ecosystem
+title: From Witnessing to Transparent Ecosystems
 topImage:
 type: standard
 room: Ziggy Stardust
 start: 2024-10-09T14:00:00
-speaker: Fillipo Valsorda
+speaker: Fillipo Valsorda, Andrea Barisani et al.
```
Comment (AlCutter) [inline content/schedule/witness.md:9]: , Martin, et Al? :)

File: content/schedule/registration.md (hunk 1 above)
Comment (tracymiranda) [reply content/schedule/registration.md:23]: Computer says no

**Review by AlCutter — APPROVED** [review]

File: content/schedule/registration.md (hunk 1 above)
Comment (AlCutter) [reply content/schedule/registration.md:23]: 🍳

**Pull request merged**